### PR TITLE
feat: add object lab (JS only)

### LINKED ISSUE #429: chore: add lab for objects in frontend cert(JS only)
The previous workshop and lecture videos will teach campers about objects.

So we need a lab for them to do immediately after that.

- let
- const
- assignment
- reassignment
- basic string usage
- console.log
- camelCase
- string concatenation using the + operator
- template literals
- str.length property
- bracket notation and retrieving characters from strings
- str.indexOf method
- Number()
- Math.floor
- Math.random
- getting a random number from a range
- prompt()
- ===
- if/else if/else
- true and false (truthy and falsey)
- comparison operators(< >=)
- logical operators (|| &&)
- array basics
- push
- pop
- shift
- unshift
- basic object usage

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/js-projects` directory
- [ ] add your completed project in an `script.js` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #429

### CODE REVIEW

File: frontend-cert/js-projects/quiz-game/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+1. You should create an object named `questions` with three keys, `science`, `geography`, and `literature`, and the values of an empty array.
```
Comment (moT01) [inline frontend-cert/js-projects/quiz-game/user-stories.md:1]:
```suggestion
1. You should create an object named `questions` with three keys, `science`, `geography`, and `literature`, each with the value of an empty array.
```

Just to make it a little more clear.

File: frontend-cert/js-projects/quiz-game/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+1. You should create at least four `question#` variables, where `#` is replaced by a number starting from `1`.
+1. Each `question#` variable should be an object with the keys `question`, `choices`, and `answer`.
+1. The `question` key should have the value of a string representing a question.
+1. The `choices` key should have the value of an array containing three strings, which are alternative aswers to the question.
```
Comment (moT01) [inline frontend-cert/js-projects/quiz-game/user-stories.md:5]:
```suggestion
1. The `choices` key should have the value of an array containing three strings, which are alternative answers to the question.
```

**Review by moT01 — COMMENTED** [review]
This is looking good @Dario-DC 👍 I left a few minor suggestions.

This one feels like it would be a good candidate to have a UI.

Also, I would add somewhere in the user stories that the "Answer" value needs to be in the choices array or something like that.

**Comment by Dario-DC** [comment]
I've implemented your suggestions @moT01 I'm not sure about the UI. Considering what campers will know at this point of the curriculum I wouldn't know how to implement it.

**Review by moT01 — APPROVED** [review]

**Pull request merged**